exasol_advanced_analytics_framework/interface/create_event_handler_udf.py

```diff
@@ -126,13 +125,19 @@ def _create_udf_context_wrapper(ctx) -> UDFContextWrapper:
 
     @staticmethod
     def _wrap_return_query(
-            iter_num: int, bucketfs_conn: str,
-            query: str, query_columns: dict) -> Tuple[str,str]:
-        columns_str = ",".join(query_columns.keys())
-        tmp_view_name = "tmp_view".upper()
-        event_handler_udf_name = "AAF_EVENT_HANDLER_UDF".upper()
-        query_create_view = f"Create view {tmp_view_name} as {query};"
-        query_event_handler = f"SELECT {event_handler_udf_name}" \
-                              f"({iter_num},'{bucketfs_conn}',{columns_str}) " \
-                              f"FROM {tmp_view_name};"
+            iter_num: int, bucketfs_conn: str, schema: str,
+            return_query: EventHandlerReturnQuery) \
+            -> Tuple[str, str]:
+        tmp_view_name = \
```

Comment:
Please, use these classes, they can handle quoting and also validate for valid identifier

https://github.com/exasol/data-science-utils-python/tree/main/exasol_data_science_utils_python/preprocessing/sql/schema